### ISSUE TITLE
`titleformat` key not recognized

### ISSUE THREAD

**Issue by mdcge** [jolars/moloch#23]
Using the minimalistic setup:

```
\documentclass{beamer}
\usetheme[titleformat=regular]{moloch}

\title{Moloch 'titleformat' problem}
\date{Today}

\begin{document}

\maketitle

\begin{frame}{A frame}
    Just some text.
\end{frame}

\end{document}
```

I get the error message `Package pgfkeys Error: I do not know the key '/titleformat', to which you passed 'regular', and I am going to ignore it. Perhaps you misspelled it.`

The same happens with any variation of the `titleformat` key: `titleformat plain`, `titleformat title`, `titleformat subtitle`, etc... Other keys such as `sectionpage` for example do work, however.

**Comment by jolars** [comment]
Thanks for the report. Which version of moloch do you have installed? (Check your `.log` file for the package version.)

`titleformat` is only available from 0.4.0 and onwards.

**Comment by mdcge** [comment]
Indeed. I'm working on overleaf and it seems that uses v0.3.0 of moloch. Thank you.
